implementation/src/main/java/io/smallrye/jwt/auth/principal/KeyLocationResolver.java

```diff
@@ -45,77 +47,113 @@
  */
 public class KeyLocationResolver implements VerificationKeyResolver {
     private static final Logger log = Logger.getLogger(KeyLocationResolver.class);
-    private String location;
     private String content;
+    private HttpsJwks httpsJwks;
```

Comment:
Would be good to add a comment that `content` and `httpsJwks` are mutually exclusive.

Reply:
OK